front/js/shared/tracking.es6

```diff
@@ -32,9 +32,16 @@
   };
 
   function setUpListeners() {
-    mediator.subscribe('onOneClickBuy', () => {
+    mediator.subscribe('onOneClickBuy', (_, phone) => {
       reachGoal('CMN_BUY_SEND');
       reachGoal('FAST_BUY_SEND');
+      // @todo #977:120m  Create some lazy() wrapper for counters.
+      //  `lazy(my_counter).reachGoal()` should asynchronous delay reaching
```

Comment:
[That](https://github.com/fidals/refarm-site/pull/314/files) GA loader could be helpful as a reference implementation